openquake/server/views.py

```diff
@@ -603,6 +603,9 @@ def aelo_run(request):
     :param request:
         a `django.http.HttpRequest` object containing lon, lat, vs30, siteid
     """
+    if not config.directory.mosaic_dir:
+        return HttpResponseServerError(
+            'mosaic_dir is not specified in openquake.cfg')
```

Comment:
No, this is not the right place. If `mosaic_dir` is not set the WebServer must not start at all.